azurerm/internal/services/apimanagement/api_management_user_resource.go

```diff
@@ -145,7 +145,7 @@ func resourceArmApiManagementUserCreateUpdate(d *schema.ResourceData, meta inter
 		properties.UserCreateParameterProperties.State = apimanagement.UserState(state)
 	}
 
-	if _, err := client.CreateOrUpdate(ctx, resourceGroup, serviceName, userId, properties, ""); err != nil {
+	if _, err := client.CreateOrUpdate(ctx, resourceGroup, serviceName, userId, properties, nil, ""); err != nil {
```

Comment:
what's this nil value here? as per the last PR this is expecting where this resource is being created/updated - so sending the Old/New Portal is misleading - presumably we should be sending `terraform` here?

Reply:
This new parameter is `notify` - I will try to pass `terraform` to `appType` in other two methods and see if it works

Reply:
@ArcturusZhang can we set this to true/false (matching the default value used in the other resources) here?

Reply:
No problem. On my way. BTW do we need to revert those changes on the apimanagement identity?